### PR TITLE
[Explorer] Adding Verified On-chain Collection Support

### PR DESCRIPTION
**Pull request by willroeder**
This change adds Verified On-chain Collection data to help consumers check if their NFTs are authentic. Note the new "Verified Collection" pill beside Master Edition and the "Verified Collection Address" addition to the Overview Section.

Screen Shot 2022-03-04 at 3 29 20 PM

### CODE REVIEW

File: explorer/src/components/account/MetaplexNFTHeader.tsx [hunk 1]
```diff
@@ -174,3 +175,16 @@ function getIsMutablePill(isMutable: boolean) {
     }`}</span>
   );
 }
+
+function getIsVerifiedCollection(isVerified?: boolean) {
+  const onchainVerifiedToolTip =
+    "This NFT has been verified as a member of an no-chain collection. This tag guarantees authenticity.";
```
Comment (samuelvanderwaal) [inline explorer/src/components/account/MetaplexNFTHeader.tsx:181]: `an *on-chain* collection

Comment (KingSoz) [reply explorer/src/components/account/MetaplexNFTHeader.tsx:181]: Good call. Thanks!

Comment (KingSoz) [reply explorer/src/components/account/MetaplexNFTHeader.tsx:181]: no-chain > on-chain

**Comment by KingSoz** [comment]
I need to add a SPL-TOKEN owner check. Will update this PR shortly.

**Comment by willroeder** [comment]
@oJshua

**Pull request merged**